piccolo/query/methods/insert.py

```diff
@@ -36,6 +51,10 @@ def returning(self: Self, *columns: Column) -> Self:
         self.returning_delegate.returning(columns)
         return self
 
+    def on_conflict(self: Self, conflict: OnConflict) -> Self:
```

Comment:
Do you think we should allow the user to pass in specific columns?

```python
await Band.insert(Band(name="Pythonistas")).on_conflict(Band.name, do_nothing=True)
```

Reply:
If you look at the tests, even now the user can update one or more columns on `do_update` conflict. For `do_nothing` conflict I don't see point of allowing specifying columns because on `do_nothing` conflict columns don't change anyway.